src/main/java/com/codeborne/selenide/impl/Randomizer.java

```diff
@@ -0,0 +1,9 @@
+package com.codeborne.selenide.impl;
+
+import java.util.UUID;
+
+public class Randomizer {
+  public String text() {
```

Comment:
Not a text but an uuid. Method signature indicates the return type is a String and in general it's a text

Reply:
Sorry, I don't understand. The intention of this method is to generate random text. The fact that it uses UUID is just an implementation detail.